src/routing_table/mod.rs

```diff
@@ -627,7 +627,7 @@ impl<T: Binary + Clone + Copy + Debug + Default + Hash + Xorable> RoutingTable<T
     //
     // Returns `true` if `dst` is a single node with name equal to `our_name`, or if `dst` is a
     // group and the closest group is our group.
-    pub fn is_recipient(&self, dst: &Destination<T>) -> bool {
+    pub fn in_authority(&self, dst: &Destination<T>) -> bool {
```

Comment:
Does it make sense to change the name here? The parameter is of type `Destination`, and the function used to check if we are in the source authority is the one in `node.rs` anyway.